breaking-front/src/components/ProfileSettingForm/ProfileSettingForm.js

```diff
@@ -86,6 +97,13 @@ export default function ProfileSettingForm({
     }
   };
 
+  const handlePhoneNumberChange = (event) => {
+    setForm((form) => ({
```

Comment:
```suggestion
    let phoneNumber= Number(event.target.value);
    if (Number.isNaN(phoneNumber)) return;
    else{
     setForm((form)=>({
     ...form,
     phoneNumber:phoneNumber
     })
     }
```

이렇게 하면 문자를 입력할때는 페이지가 랜더링이 되지 않습니다.